### DISCUSSION TITLE
Chromedriver 91 needs an update

### DISCUSSION THREAD

**Discussion by BKu-1** [npm/feedback#436] [General]
Hi,
it would be really great, if you could update the chromedriver 91 to 91.0.4472.101, as the latest has a major issue that occurs in test automation.

https://www.npmjs.com/package/chromedriver/v/91.0.0

ChromeDriver 91.0.4472.101
Supports Chrome version 91
        Resolved issue 1205107: Unable to get attribute value for fields on the web application.

Thank you a lot.
BKu-1

**Comment by MylesBorins** [comment]
Hey @BKu-1 . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support for packages on the registry. Please open an issue on the repository for this package if you would like support

https://github.com/giggio/node-chromedriver
